pendulum/datetime.py

```diff
@@ -571,7 +571,7 @@ def is_same_day(self, dt):
 
         return self.to_date_string() == dt.to_date_string()
 
-    def is_birthday(self, dt=None):
+    def is_anniversary(self, dt=None):
```

Comment:
docstring needs updating

Reply:
You are definitely right 👍 I will update, thanks!